frontend/packages/portal-frontend/src/data-explorer-2/components/ContextManager/DownloadContextModal.tsx

```diff
@@ -47,18 +42,23 @@ function DownloadContextModal({
     }
 
     fetchContext(contextHash)
-      .then(evaluateContext)
-      .then((evaluated) => {
-        let labels = evaluated.labels;
-
+      .then(fetchContextLabels)
+      .then(async (depmap_ids) => {
+        // For now, we assume that the downloads are always depmap_models
+        // And the actual labels need to be loaded with a separate request
```

Comment:
This is not really true. It works with other types like genes but those other types don't have a concept of display name.